documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -7,6 +7,8 @@
 * (IDETECT-3641) Resolved scan failure caused by leading/trailing spaces specified within quotes for `detect.project.name` or `detect.project.version.name` properties.
 * (IDETECT-3776) Resolved an issue with not detecting certain components in `go.mod` files as transitive dependencies when marked with `// indirect`, by improving identification of direct and indirect dependencies.
 * (IDETECT-3817) Improved handling of large inspection results to prevent OutOfMemory exceptions and optimize memory usage.
+* (IDETECT-3888) Improved runtime performance PIP Inspector in the case of aws-cdk dependencies by passing package history list by reference instead of value.
```

Comment:
Minor tweaks suggestions:

* (IDETECT-3888) Improved the runtime performance of PIP Inspector for aws-cdk dependency cases by passing the package history list by reference instead of value.
* (IDETECT-3867) Resolved a lack of support for properties set in SPRING_APPLICATION_JSON environment variable for configuring [solution_name] when the Self Update feature is utilized.

Reply:
Done, thanks!